### ISSUE TITLE
Prepare for public release

### ISSUE THREAD

**Issue by boulund** [ctmrbio/BACTpipe#76]
There are a few things to take care of before we can make the repository entirely public:

- [x] Remove milou stuff
- [x] Fix the final details in the most important outstanding issues, e.g. #60, which is practically done already.
- [x] Create a [Release](https://help.github.com/articles/creating-releases/) for BACTpipe version 2.5.3 (or whatever version we're currently on when we're making the release).
- [x] Stop committing directly to the `master` branch. As soon as we have made the repository public it is important that we avoid making commits directly to the master branch, as that makes it harder to troubleshoot issues that people might experience, and it is easier for us to keep a `develop` branch (or something) that can accumulate all updates incorporated into new releases. It also reduces the risk of us accidentally making changes without bumping the version number. Having several different versions of the code with different version numbers publicly available is a mess, so let's all help out to prevent that! :D 
- [x] Create a [changelog](https://keepachangelog.com/en/1.0.0/) before our next release. The changelog should be created in the `develop` branch and merged into the `master` branch as soon as we make a new release. It makes it easier for users to see what has changed, to determine if it's worth upgrading.

**Comment by boulund** [comment]
I should perhaps mention that the documentation at https://bactpipe.readthedocs.org now has as selector down at the bottom left so you can look at the documentation of the `develop` branch separately from what is the default (i.e. the `master` branch). This makes it easier to make sure the docs in the develop branch look OK before merging.
